controllers/storageclient_controller.go

```diff
@@ -549,6 +572,56 @@ func (r *StorageClientReconciler) reconcileSharedfileStorageClaim() error {
 	return nil
 }
 
+func (r *StorageClientReconciler) reconcileRemoteNoobaa() error {
+	// Create noobaa CR referencing the join secret
+	nb := &nbv1.NooBaa{}
+	nb.Name = "noobaa-remote"
+	nb.Namespace = r.OperatorNamespace
+	nb.Spec.JoinSecret = &corev1.SecretReference{
+		Name:      joinSecretName,
```

Comment:
Could we append the storageClient hash to noobaa and joinSecret, this will ensure if we move to support multiple storageClients in the same cluster we won't run into name collision

Reply:
Sure, I will take it up in a different PR
I have raised an issue for the same
https://github.com/red-hat-storage/ocs-client-operator/issues/189